ui/tests/integration/components/topo-viz/datacenter-test.js

```diff
@@ -119,15 +119,15 @@ module('Integration | Component | TopoViz::Datacenter', function (hooks) {
     assert.ok(TopoVizDatacenter.label.includes(`${allocs.length} Allocs`));
     assert.ok(
       TopoVizDatacenter.label.includes(
-        `${formatBytes(memoryReserved, 'MiB')}/${formatBytes(
+        `${formatBytes(memoryReserved, 'MiB')} / ${formatBytes(
           memoryTotal,
           'MiB'
         )}`
       )
     );
     assert.ok(
       TopoVizDatacenter.label.includes(
-        `${formatHertz(cpuReserved, 'MHz')}/${formatHertz(cpuTotal, 'MHz')}`
+        `${formatHertz(cpuReserved, 'MHz')} / ${formatHertz(cpuTotal, 'MHz')}`
```

Comment:
Side-effect of adding the tooltips, which meant this text is now split between two `<span>` so there's a space.

Reply:
Extracted to https://github.com/hashicorp/nomad/pull/17647